exporter/exporter.go

```diff
@@ -41,10 +41,11 @@ type Exporter struct {
 	attachedProgs            map[string]map[*libbpfgo.BPFProg]bool
 	descs                    map[string]map[string]*prometheus.Desc
 	decoders                 *decoder.Set
+    btfFilePath				 string
```

Comment:
Normally this is done by your editor on save, but you can also enforce formatting by running:

```
go fmt exporter/exporter.go
```

Reply:
Thanks.